src/hooks/usePullRequestQuickstart.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+
+import { getQuickstartFilesFromPR } from '../utils/preview/fetchHelpers';
+import { parseQuickstartFilesFromPR } from '../utils/preview/parseHelpers';
+import { navigate } from 'gatsby';
+
+const usePullRequestQuickstart = (location) => {
+  const [contentFiles, setContentFiles] = useState([]);
```

Comment:
```suggestion
  const [quickstart, setQuickstart] = useState({});
```
Since the hook has parsing now, we might want to change this to hold the state of the parsed quickstart instead since we don't need to worry about the files anymore.